pkg/snmp/mibs/ups_mib/ups_alarms_table.go

```diff
@@ -11,7 +11,7 @@ import (
 // upsAlarmsInfo is what the device info will be under SNMP OID
 // .1.3.6.1.2.1.33.1.6.3 view it in a MIB browser to see the names.
 var upsAlarmsInfo = []string{
-	"upsAlarmBateryBad",
+	"upsAlarmBatteryBad",
```

Comment:
I see it! My bad for not catching this - innocuous enough that little bugger

Reply:
Looks like you'll also need to bump the test table string

https://github.com/vapor-ware/synse-snmp-plugin/pull/68/files#diff-d9b867eec6a15085986cf48201d1c40ed97e53001010b2af59e252ab8c244fd1R16

Line 16 in the test suite

Reply:
Updated - thanks. I need to slow down today and be more careful.